docs/Module - CI.md

```diff
@@ -1,26 +1,112 @@
+4.  Code Analysis : Scans for misconfigurations and secrets using the [`module-ci-code-analysis.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-code-analysis.yaml) workflow
+5.  Test : Runs Terraform tests in various environments via the [`module-ci-test.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-test.yaml) workflow
+6.  Issue Creation : Creates GitHub issues for failed workflows by calling the [`module-ci-issue.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-issue.yaml) workflow
+
+---
+
+## Module CI Validate 
+ File : [`.github/workflows/module-ci-validate.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-validate.yaml)
+
+### Purpose 
+This workflow validates Terraform configurations to ensure they are syntactically correct, formatted properly, and meet validation standards.
+
+### Steps 
+1.  Repository Checkout : Clones the repository for validation
+2.  Fetch Terraform Version : Determines the required Terraform version using a custom action
+3.  Terraform Setup : Installs and configures Terraform based on the required version
+4.  Terraform Initialization : Runs `terraform init` with `-backend=false` to skip backend configuration
+5.  Format Check : Ensures consistent code style using `terraform fmt`
+6.  Validation : Checks the validity of Terraform configurations using `terraform validate`
+
+---
+
+## Module CI Lint 
+ File : [`.github/workflows/module-ci-lint.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-lint.yaml)
+
+### Purpose 
+The `module-ci-lint` workflow ensures that Terraform configurations adhere to best practices using TFLint, a linter specifically designed for Terraform.
+
+### Steps 
+1.  Repository Checkout : Clones the repository for linting
+2.  Fetch Terraform Version : Identifies the Terraform version needed for linting
+3.  Terraform Setup : Installs and configures Terraform for compatibility with TFLint
+4.  TFLint Configuration Sync : Pulls TFLint configuration files from the governance repository to ensure consistency across projects
+5.  TFLint Setup : Installs and initializes TFLint, including plugins
+6.  Lint Execution : Runs TFLint checks on Terraform configurations, including module examples, to ensure compliance with defined rules
+
+---
+
+## Module CI Documentation 
+ File : [`.github/workflows/module-ci-documentation.yaml`](https://github.com/cloudeteer/terraform-governance/blob/main/.github/workflows/module-ci-documentation.yaml)
+
+### Purpose 
+This workflow automates the generation and validation of Terraform module documentation using `terraform-docs`. It ensures that documentation is consistent with the module's actual configuration.
```

Comment:
The documentation generation isn't automated by this workflow, it's validated by it.

```suggestion
This workflow validates the Terraform module documentation using `terraform-docs`. It ensures that documentation is consistent with the module's actual configuration.
```